tests/providers/ssh/operators/test_ssh.py

```diff
@@ -138,7 +160,9 @@ def test_no_output_command(self):
     def test_arg_checking(self):
```

Comment:
I'm wondering if we should split this apart? We now have 7 different cases in a single test.

Reply:
If I understand what github is telling me, it looks like I can't be the one to resolve the conflict.

If I have some time tonight I'll split this test apart. I'm also happy to open a new PR for it if we want to close this one - whatever is easier for you all.

Reply:
For the conflict, you should be able to resolve it by rebasing on the current main branch.

Reply:
Alright, I think it's good now. I rebased and merged my changes to the file back in and there are no more conflicts.